### PR TITLE
2.14.17 Release Notes

### PR DESCRIPTION
**Pull request by desertaxle**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed.

For new functions or classes in the Python SDK:

- [ ] This pull request includes helpful docstrings.
- [ ] If a new Python file was added, this pull request contains a stub page in the Python SDK docs and an entry in `mkdocs.yml` navigation.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,117 @@
...
+```
+
+</details>
+
+This feature is experimental and subject to change. Please try it out and let us know what you think!
+
+See [the PR](https://github.com/PrefectHQ/prefect/pull/11476) for implementation details.
+
+### Enhancements
+- Add `url` to `runtime.flow_run` — https://github.com/PrefectHQ/prefect/pull/11686
```
Comment (serinamarie) [inline RELEASE-NOTES.md:87]:
```suggestion
- Add `url` to `prefect.runtime.flow_run` — https://github.com/PrefectHQ/prefect/pull/11686
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,117 @@
...
+- Add ability to subpath the `/ui-settings` endpoint — https://github.com/PrefectHQ/prefect/pull/11701
+
+### Fixes
+- Handle `pydantic` v2 types in schema generation for flow parameters — https://github.com/PrefectHQ/prefect/pull/11656
+- Increase flow run resiliency by gracefully handling PENDING to PENDING state transitions — https://github.com/PrefectHQ/prefect/pull/11695
```
Comment (serinamarie) [inline RELEASE-NOTES.md:92]:
```suggestion
- Increase flow run resiliency by gracefully handling `PENDING` to `PENDING` state transitions — https://github.com/PrefectHQ/prefect/pull/11695
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,117 @@
...
+
+### Experimental
+- Add endpoint to `Runner` web server to allow running an flow available to a `Runner` instance — https://github.com/PrefectHQ/prefect/pull/11476
```
Comment (serinamarie) [inline RELEASE-NOTES.md:95]: I think we can remove this section now

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,117 @@
...
+- Fix typo in automations guide — https://github.com/PrefectHQ/prefect/pull/11716
+- Add Events and remove Incidents from Concepts index page — https://github.com/PrefectHQ/prefect/pull/11708
+- Remove subflow task tag concurrency warning — https://github.com/PrefectHQ/prefect/pull/11725
+- Remove line from `Pausing a flow` docs saying you can `pause` from the UI — https://github.com/PrefectHQ/prefect/pull/11730
+- Update wording for Jinja templating guide in automations concept doc for readability — https://github.com/PrefectHQ/prefect/pull/11729
+- Fix broken links — https://github.com/PrefectHQ/prefect/pull/11692
+- Fix typo in flows concept doc — https://github.com/PrefectHQ/prefect/pull/11693
+
+### Contributors
+
```
Comment (serinamarie) [inline RELEASE-NOTES.md:110]:
```suggestion
```
extra line

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,117 @@
...
+- Handle `pydantic` v2 types in schema generation for flow parameters — https://github.com/PrefectHQ/prefect/pull/11656
+- Increase flow run resiliency by gracefully handling PENDING to PENDING state transitions — https://github.com/PrefectHQ/prefect/pull/11695
+
+### Experimental
+- Add endpoint to `Runner` web server to allow running an flow available to a `Runner` instance — https://github.com/PrefectHQ/prefect/pull/11476
+
+### Documentation
+- Add documentation for `cache_result_in_memory` argument for `flow` decorator — https://github.com/PrefectHQ/prefect/pull/11669
+- Add runnable example of `flow.from_source()` — https://github.com/PrefectHQ/prefect/pull/11690
+- Make Creating interactive workflows guide easier to find via search — https://github.com/PrefectHQ/prefect/pull/11704
```
Comment (serinamarie) [inline RELEASE-NOTES.md:100]:
```suggestion
- Improve discoverability of creating interactive workflows guide — https://github.com/PrefectHQ/prefect/pull/11704
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,117 @@
...
+- Fix typo in automations guide — https://github.com/PrefectHQ/prefect/pull/11716
+- Add Events and remove Incidents from Concepts index page — https://github.com/PrefectHQ/prefect/pull/11708
```
Comment (serinamarie) [inline RELEASE-NOTES.md:102]:
```suggestion
- Remove events and incidents from concepts index page — https://github.com/PrefectHQ/prefect/pull/11708
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,117 @@
...
+- Remove subflow task tag concurrency warning — https://github.com/PrefectHQ/prefect/pull/11725
+- Remove line from `Pausing a flow` docs saying you can `pause` from the UI — https://github.com/PrefectHQ/prefect/pull/11730
```
Comment (serinamarie) [inline RELEASE-NOTES.md:104]:
```suggestion
- Remove misleading line on pausing a flow run from the UI — https://github.com/PrefectHQ/prefect/pull/11730
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,117 @@
...
+- Update wording for Jinja templating guide in automations concept doc for readability — https://github.com/PrefectHQ/prefect/pull/11729
```
Comment (serinamarie) [inline RELEASE-NOTES.md:105]:
```suggestion
- Improve readability of Jinja templating guide in automations concept doc — https://github.com/PrefectHQ/prefect/pull/11729
```

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,117 @@
...
+- Fix broken links — https://github.com/PrefectHQ/prefect/pull/11692
```
Comment (serinamarie) [inline RELEASE-NOTES.md:106]:
```suggestion
- Resolve links to relocated interactive workflows guide — https://github.com/PrefectHQ/prefect/pull/11692
```

**Review by urimandujano — APPROVED** [review]
Looks good!

**Review by serinamarie — APPROVED** [review]
LGTM!

**Pull request merged**